tutorials/R/Hyperspectral/Intro-hyperspectral/Work-With-Hyperspectral-Data-In-R/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -416,7 +426,7 @@ Remember that the metadata for the `Reflectance` dataset designated -9999 as
 `data ignore value`. Thus, let's set all pixels with a value == -9999 to `NA` 
 (no value). If we do this, R won't try to render these pixels.
 
-```{r set-values-NA}
+```{r set-values-NA, fig.cap="Plot of reflectance values for band 9 data with values equal to -9999 set to NA."}
```

Comment:
Please describe the shape here, too. How does this improve the information content of the histogram?